packages/blockchain/flow/src/lib/flow.sdk.spec.ts

```diff
@@ -0,0 +1,265 @@
+import { TatumFlowSDK } from './flow.sdk'
+import {
+  BlockchainTestData,
+  REPLACE_ME_WITH_TATUM_API_KEY,
+  TEST_DATA,
+  walletTestFactory,
+} from '@tatumio/shared-testing'
+import { SdkWithWalletFunctions } from '@tatumio/shared-blockchain-abstract'
+import {
+  ChainFlowBurnNft,
+  ChainFlowMintMultipleNft,
+  ChainFlowMintNft,
+  ChainFlowTransferNft,
+  Currency,
+  TransferFlow,
+  TransferFlowCustomTx,
+} from '@tatumio/shared-core'
+import dedent from 'dedent-js'
+
+describe('TatumFlowSDK', () => {
+  jest.setTimeout(999999)
+
+  const sdk = TatumFlowSDK({ apiKey: REPLACE_ME_WITH_TATUM_API_KEY })
+
+  describe('Wallet', () => {
+    describe('Generate wallet', () => {
+      walletTestFactory.generateBlockchainWallet(sdk.wallet, TEST_DATA.FLOW)
+    })
+
+    describe('Address from XPUB', () => {
+      walletTestFactory.generateAddressFromXpub(sdk.wallet, TEST_DATA.FLOW)
+    })
+
+    describe('Private key from mnemonic', () => {
+      walletTestFactory.generatePrivateKeyFromMnemonic(sdk.wallet, TEST_DATA.FLOW)
+    })
+
+    describe('Address from private key', () => {
+      walletTestFactory.generateAddressFromPrivateKey = (
```

Comment:
what is this?

Reply:
Wallet tests for Flow are almost the same as for other chains except this one method so I reused it and removed two unsuitable tests.